node/src/manager/commands/rewind.rs

```diff
@@ -116,8 +128,22 @@ pub async fn run(
     println!("\nRewinding deployments");
     for deployment in &deployments {
         let loc = deployment.locator();
-        subgraph_store.rewind(loc.hash.clone(), block_ptr_to.clone())?;
-        println!("  ... rewound {}", loc);
+        let deployment_details = subgraph_store.load_deployment_by_id(loc.clone().into())?;
+
+        let block_ptr_to = block_ptr_to.clone();
+
+        if let Some(block_ptr) = block_ptr_to {
+            subgraph_store.rewind(loc.hash.clone(), block_ptr.clone())?;
+            println!("  ... rewound {}", loc);
+        } else {
+            if let Some(block_ptr) = deployment_details.start_block {
```

Comment:
If the deployment doesn't have a start block, shouldn't we use the genesis block as the point to which we truncate?